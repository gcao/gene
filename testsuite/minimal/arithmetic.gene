# Minimal arithmetic test
(print (+ 2 3))
(print (- 7 4))
(print (* 6 7))
(print (/ 15 3))