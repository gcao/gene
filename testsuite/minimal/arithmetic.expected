5
3
42
5.0